src/DotNetWorker/Invocation/DefaultFunctionExecutor.cs

```diff
@@ -28,22 +28,20 @@ public async Task ExecuteAsync(FunctionContext context)
                 _ => _invokerFactory.Create(context.FunctionDefinition));
 
             object? instance = invoker.CreateInstance(context.InstanceServices);
-            var bindingFeature = context.Features.Get<IModelBindingFeature>();
+            var modelBindingFeature = context.Features.Get<IModelBindingFeature>();
 
             object?[] inputArguments;
-            if (bindingFeature is null)
+            if (modelBindingFeature is null)
             {
-                Log.FunctionBindingFeatureUnavailable(_logger, context);
+                Log.ModelBindingFeatureUnavailable(_logger, context);
                 inputArguments = new object?[context.FunctionDefinition.Parameters.Length];
             }
             else
             {
-                inputArguments = bindingFeature.BindFunctionInput(context);
+                inputArguments = modelBindingFeature.BindFunctionInput(context);
             }
 
-            object? result = await invoker.InvokeAsync(instance, inputArguments);
-
-            context.InvocationResult = result;
+            context.GetBindings().InvocationResult = await invoker.InvokeAsync(instance, inputArguments);
```

Comment:
nit: We should do the same thing we're doing for model bindings: pull the function bindings feature, validate it was present and use the feature directly.

Reply:
`GetBindings()` will throw if it's not registered. I really wasn't sure what we'd do if it wasn't there...